src/WebSdk/ProjectSystem/Targets/Microsoft.NET.Sdk.Web.DefaultItems.props

```
@@ -0,0 +1,44 @@
+<!--
+***********************************************************************************************
+Microsoft.NET.Sdk.Web.DefaultItems.props
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (c) .NET Foundation. All rights reserved.
+***********************************************************************************************
+-->
+<Project ToolsVersion="14.0">
+
+  <ItemGroup Condition="'$(EnableDefaultItems)' == 'true' 
+             And '$(EnableDefaultContentItems)' == 'true' 
+             And '$(EnableDefaultNoneItems)' == 'true'
+             And '$(AppDesignerFolder)' != ''">
+
+    <_webToolingArtifacts Include="$(AppDesignerFolder)\launchSettings.json;
```

Nit: _
```suggestion
    <_WebToolingArtifacts Include="$(AppDesignerFolder)\launchSettings.json;
```

A little unusual to see lower cased property names

---

Did a couple of renames before I landed at this name and messed up the casing in the end. Thanks for catching it. Will fix it.